common/components/ui/Modal.tsx

```diff
@@ -48,19 +43,19 @@ export default class Modal extends Component<Props, {}> {
     const hasButtons = buttons && buttons.length;
 
     return (
-      <div>
+      <div onClick={this.handleClickOutside}>
```

Comment:
Taylor explicitly did not want this behavior in modals anymore from v3.

Reply:
I ported this after seeing the design on V3 but will remove